webapp/tests/karma/ts/services/contact-view-model-generator.service.spec.ts

```diff
@@ -14,99 +21,104 @@ describe('ContactViewModelGenerator service', () => {
+  const stubGetDataRecords = (dataRecords) => {
+    getDataRecords.resolves(dataRecords);
   };
 
   beforeEach(() => {
-    module('inboxApp');
-    module($provide => {
-      search = sinon.stub();
-      dbGet = sinon.stub();
-      dbQuery = sinon.stub();
-      dbAllDocs = sinon.stub();
-      const types = [
-        { id: 'family' },
-        { id: 'person', sort_by_dob: true, icon: childPlaceIcon, person: true, parents: [ 'family', 'clinic' ] },
-        { id: 'chp', person: true, parents: [ 'mushroom' ] },
-        { id: 'red-herring' },
-        { id: 'clinic', parents: [ 'mushroom' ] },
-        { id: 'mushroom', name_key: 'label.mushroom' }
-      ];
-      ContactTypes = { getAll: sinon.stub().resolves(types) };
-      lineageModelGenerator = { contact: sinon.stub() };
-      GetDataRecords = sinon.stub();
-      Session = { isOnlineOnly: function() {} };
-
-      $provide.factory('DB', KarmaUtils.mockDB({ get: dbGet, query: dbQuery, allDocs: dbAllDocs }));
-      $provide.value('Search', search);
-      $provide.value('ContactTypes', ContactTypes);
-      $provide.value('$q', Q); // bypass $q so we don't have to digest
-      $provide.value('LineageModelGenerator', lineageModelGenerator);
-      $provide.value('GetDataRecords', GetDataRecords);
-      $provide.value('Session', Session);
-
-      const parentId = 'districtsdistrict';
-      const contactId = 'mario';
-      childContactPerson = { _id: contactId, name: 'sandy', type: 'person', parent: { _id: parentId } };
-      deceasedChildPerson = {
-        _id: 'deceaseduuid',
-        name: 'casper',
-        type: 'person',
-        date_of_death: 123456789,
-        parent: { _id: parentId }
-      };
-
-      childPerson = { _id: 'peach', type: 'person', name: 'Peach', date_of_birth: '1986-01-01' };
-
-      childPerson2 = { _id: 'zelda', type: 'person', name: 'Zelda', date_of_birth: '1985-01-01' };
-      childPlace = { _id: 'happyplace', type: 'mushroom', name: 'Happy Place', contact: { _id: contactId } };
-      childPlace2 = { _id: 'happyplace2', type: 'mushroom', name: 'Happy Place 2' };
-
-      doc = {
-        _id: parentId,
-        type: 'clinic',
-        contact: { _id: contactId }
-      };
-      forms = [];
-    });
-    inject(_ContactViewModelGenerator_ => service = _ContactViewModelGenerator_);
+    search = sinon.stub();
+    dbGet = sinon.stub();
+    dbQuery = sinon.stub();
+    // dbAllDocs = sinon.stub();
+    getDataRecords = sinon.stub();
+    const types = [
+      { id: 'family' },
+      { id: 'person', sort_by_dob: true, icon: childPlaceIcon, person: true, parents: [ 'family', 'clinic' ] },
+      { id: 'chp', person: true, parents: [ 'mushroom' ] },
+      { id: 'red-herring' },
+      { id: 'clinic', parents: [ 'mushroom' ] },
+      { id: 'mushroom', name_key: 'label.mushroom' }
+    ];
+    lineageModelGenerator = { contact: sinon.stub() };
+    Session = { isOnlineOnly: function() {} };
```

Comment:
Oh this variable name isn't in camelCase
Also since it refers to a service, then I think it's better to use the service name: sessionService, searchService, etc, it's going to be clearer that it's service